tests/Doctrine/Tests/Common/Proxy/LazyLoadableObjectWithTypehints.php

```diff
@@ -90,5 +93,10 @@ public function getIdentifierFieldReturnClassOneLetter(): A
         return $this->identifierFieldReturnClassOneLetter;
     }
 
+    public function getIdentifierFieldReturnClassOneLetterNullable(): ?A
```

Comment:
This one probably needs to be moved to a separate class, to avoid having the parser choke when importing the test assets on pre-7.1